css/client.css

```diff
@@ -638,3 +638,111 @@ input:checked + .slider:before {
 .tabHeader {
     background: var(--medDark);
 }
+
+/* style for wordlinks */
+.termHeader {
+    display: flex;
+    align-items: center;
+}
+.termHeader h2 {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    line-height: 38px;
+    text-transform: capitalize;
+}
+
+.termHeader  a {
+    margin:0 10px 10px;
+    color: var(--medDark);
+}
+.termHeader a:hover {
+    color: #337ab7;
+}
+
+.termExplanation {
+    /* border: 1px solid #d3d3d3; */
+    max-height: 262px;
+    height: calc(100vh);
+    overflow-y: scroll;
+    scroll-behavior: smooth;
+    scrollbar-gutter: unset;
+}
+.termExplanation b {
+    text-transform: uppercase;
+}
+
+.term-list .list-group {
+    max-height: 236px;
+    overflow-y: scroll;
+    scroll-behavior: smooth;
+    scrollbar-gutter: unset;
+    height: 100%;
+    border-top: .5px solid #ddd;
+    border-bottom: .5px solid #ddd;
+    margin-top: 15px;
+}
+
+.list-group-item {
+    flex: 1;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    text-transform: capitalize;
+}
+
+.termList {
+    display: flex;
+    flex-direction: column;
+    height: fit-content;
+    max-height: 310px;
+}
+
+.termList .form-control {
+    margin: 10px 0 15px;
+    height: 35px;
+    line-height: 35px;
+}
+.relatedTermsList {
+    padding-left: 0px;
+    list-style: none;
+}
+.relatedTermsList li {
+    display: inline;
+    padding: 0 1px;
+}
+
+.alternateTerms {
+    padding-left: 1px;
+    list-style: none;
+    font-weight: 500;
+}
+.alternateTerms li {
+    display: inline;
+}
+.relatedTermsList li:after,
+.alternateTerms li:after {
+    content: ", ";
+}
+
+.relatedTermsList li:last-child:after,
+.alternateTerms li:last-child:after {
+    content: "";
+}
+.otherLanguageExamplesList {
+    padding-left: 20px;
+}
+a {
+    cursor: pointer !important;
```

Comment:
Is this "!important" really necessary? Do the other links in the site not use pointer for links? I agree that links should be pointer, but let's keep the existing style.